src/plugins/BehaviorSpecGenerator/test.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
I now see those were already merge into the master. 
However you haven't added any tests to check the new code..